src/db/models/categoryGroup.ts

```diff
@@ -28,6 +30,7 @@ export const CATEGORY_GROUP_TYPE = t.keyof({
   reportChannel: null,
   responseType: null,
   sectorIASC: null,
+  subsetOfOrganization: null,
```

Comment:
[As I wrote](https://github.com/UN-OCHA/hpc_service/pull/3351#discussion_r1733508057) in my review of related PR in hpc_service, I don't think we should add this category group at all.